### DISCUSSION TITLE
Change the look of selected stuff in the taskbar

### DISCUSSION THREAD

**Discussion by Cornholio71** [valinet/ExplorerPatcher#3140] [Q&A]
I want this look back:
 ![this](https://github.com/valinet/ExplorerPatcher/assets/167182240/228197d0-02a7-4da6-aab6-59de87a1b47f)

Not this:
 ![not this](https://github.com/valinet/ExplorerPatcher/assets/167182240/e51c4728-75db-4d3e-a4b5-f989704800a8)
How do i change that?

**Comment by Sand216** [comment]
Do you mean hiding the labels of taskbar buttons?

In taskbar properties you can change the "Combine taskbar icons on primary taskbar" setting to "Always combine, hide labels"

Screenshot from ExplorerPatcher Wiki:

![image](https://github.com/valinet/ExplorerPatcher/assets/164842290/c712c361-9acd-4cb1-bd3d-2751dcce0655)

**Reply by Cornholio71** [reply]
That did it,thank you very much 🙏
